sopel/modules/reddit.py

```diff
@@ -52,59 +53,59 @@ def rpost_info(bot, trigger, match):
             client_id='6EiphT6SSQq7FQ',
             client_secret=None,
         )
-        s = r.submission(id=match.group(1))
-    except Exception:
-        r = praw.Reddit(user_agent=USER_AGENT)
-        s = r.get_submission(submission_id=match.group(1))
-
-    message = ('[REDDIT] {title} {link}{nsfw} | {points} points ({percent}) | '
-               '{comments} comments | Posted by {author} | '
-               'Created at {created}')
-
-    subreddit = s.subreddit.display_name
-    if s.is_self:
-        link = '(self.{})'.format(subreddit)
-    else:
-        link = '({}) to r/{}'.format(s.url, subreddit)
+        s = r.submission(id=match.group(2))
```

Comment:
Yeah, there's no group 2 any more for posts. In `post_url` and `short_post_url` only group 1 is defined (captures the submission ID). Group 1 used to be the subreddit I think, but that was useless so #1503 removed it when adding short URL support. Oh wait, that was also your PR…

Reply:
Don't use my own logic against me :D